ansible/custom_roles/mysql/defaults/main.yml

```diff
@@ -81,7 +81,7 @@ mysql_innodb_file_per_table: "1"
 # Set .._buffer_pool_size up to 80% of RAM but beware of setting too high.
 mysql_innodb_buffer_pool_size: "256M"
 # Set .._log_file_size to 25% of buffer pool size.
-mysql_innodb_log_file_size: "64M"
+#mysql_innodb_log_file_size: "64M"
```

Comment:
I believe this will also be applied to production when we roll it out. 

Is the staging version of mysql the same as staging right now?

Reply:
MySQL version is same in staging and in production. Had a warning in the logs that this is deprecated and will be removed in future releases so I commented it out.